app/controllers/admin/buildings_controller.rb

```diff
@@ -103,11 +103,12 @@ def build_building
     end
 
     def find_buildings
+      @current_state = params[:state] || "not_contacted"
```

Comment:
Is this not available on `@buildings`?

Reply:
I've tried updating Browseable to include a `default_facet` (that would be `:not_contacted`) but that in turn will confuse code that relies on the default facet being `:all`. The DeltaExporter for instance:

```
[7] pry(#<RSpec::ExampleGroups::DeltaExporter::Mapping>)> Building.all.map(&:uprn)=> ["1234567890"]
[8] pry(#<RSpec::ExampleGroups::DeltaExporter::Mapping>)> Building.search(uprn: "1234567890").count
Building.search(uprn: "1234567890").count
[8] pry(#<RSpec::ExampleGroups::DeltaExporter::Mapping>)> Building.search(uprn: "1234567890").count=> 0
```

I know the fix isn't too far away but given the current solution isn't a gross hack I'm going to keep that and finish the other features before revisiting this.